izumi-reflect/izumi-reflect/src/main/scala_2/izumi/reflect/macrortti/LightTypeTagImpl.scala

```diff
@@ -516,7 +516,13 @@ final class LightTypeTagImpl[U <: Universe with Singleton](val u: U, withCache:
             case UniRefinement(_, _) =>
               None
             case o =>
-              fromRef(o)
+              val tsym = tpef.typeSymbol
+              fromRef {
```

Comment:
@IndiscriminateCoding Hi, could you check if `fromRef(tpef.typeSymbol.definingClass)` might work instead of looping?

Reply:
Hi @neko-kai , using `definingClass` gives me compilation error:

> value definingClass is not a member of LightTypeTagImpl.this.u.Symbol

Reply:
@IndiscriminateCoding Hmm, what if you try `tpef.typeSymbol.owner`?

Reply:
@neko-kai that works! Fixed